### ISSUE TITLE
Add diverging scale handling NAs

### ISSUE THREAD

**Issue by petrbouchal** [scholaempirica/reschola#9]

**Comment by jaromir-mazak** [comment]
@petrbouchal  toto je ten diverging chart, co řešíme v jiném issue, žejo? pokud ano, tak tohle asi lze zavřít.

**Comment by petrbouchal** [comment]
Tohle je jen paleta/scale, takže součást #21, ale výsledek by měl existovat zvlášť a být použitelný i jinde, takže zatím nechávám.

**Comment by jaromir-mazak** [comment]
aha, OK, chápu.

ať to je na jednom místě, dám sem nějaké odkazy:

pro černobílý tisk:
https://www.colorcombos.com/black-and-white-color-schemes.html

obecně na barvy:
https://medialab.github.io/iwanthue/
https://color.adobe.com/cs/create/color-wheel/

**Comment by petrbouchal** [comment]
Teď přemýšlím/experimentuju nad těmi černobílými a je tam myslím tradeoff: jednou z chtěných vlastností palety je, aby barvy měly podobnou světelnost a saturaci, tj. aby některé nerazily proti ostatním nebo nesplývaly s pozadím. Pokud bychom barvy optimalizovali pro černobílý tisk, tak toto ztratíme a ty barevné palety budou v barvě vypadat divně.

Ještě to prozkoumám, ale optimální řešení asi bude pro černobílý tisk prostě použít černobílou paletu z ggplot2 (scale_colour_bw()).

U barev samotných bych raději optimalizoval pro poruchy barvocitu než pro černobílý tisk; ty poruchy má možná větší procento lidí než to bude číst černobíle vytištěné :)

**Comment by jaromir-mazak** [comment]
souhlasím, raději ať to dobře vypadá barevné elektronicky (plus to funguje pro poruchy barvocitu). jak píšeš, ten černobílý tisk můžeme řešit cíleně, pokud by byl někdy někde potřeba pomocí scale_colour_bw().
